cron/cron.sh

```diff
@@ -17,7 +17,7 @@ SOURCE="${BASH_SOURCE[0]}"
 input=$(dirname "$SOURCE")/projects.txt
 output=$(date +"%m-%d-%Y").json
 touch "$output"
-curl https://raw.githubusercontent.com/ossf/scorecard/main/cron/projects.txt >projects.txt
+echo "{ \"results\": [" >> "$output"
```

Comment:
probably want to bring back the curl line ?

Reply:
I realized the `main` branch will always have the `projects.txt` and it was redundant.

Reply:
latest version right?, makes sense

Reply:
The cloudbuild runs on merge to main.

 The cron uses the latest image from the cloudbuild.  So it will have the latest projects.txt. So the latest project.txt will be the same in the repo and also be the same inside the container.